dashboard/test/fixtures/plc_courses.yml

```diff
@@ -0,0 +1,4 @@
+Exploring Computer Science PD Course:
+  name: "Exploring Computer Science"
+  created_at: 2015-10-16 22:36:38.000000000 Z
+  updated_at: 2015-10-16 22:36:38.000000000 Z
```

Comment:
So this was because we had a [Plc::CourseUnit](https://github.com/code-dot-org/code-dot-org/blob/d655cac9f3167978d3383d41dcb7f0ec0f9c27b6/dashboard/test/fixtures/plc_course_units.yml) fixture but no associated `Plc::Course`? Is that a state we can get into naturally? Is it worth protecting against that state in it happens in the real world, or is this just a result of incomplete fixture data?